modules/spring-config/src/main/resources/META-INF/mule-module.xsd

```diff
@@ -216,7 +216,7 @@
         <xsd:complexContent>
             <xsd:extension base="mule:abstractExtensionType">
                 <xsd:sequence maxOccurs="unbounded" minOccurs="1">
-                    <xsd:choice minOccurs="1" maxOccurs="unbounded">
+                    <xsd:choice minOccurs="0" maxOccurs="unbounded">
```

Comment:
this is enabling the very hackish

Reply:
the same amount of hackish as the Java SDK, so it's a bug on the XML SDK for not behaving the same:
```
package org.mule.extension.internal;

import org.mule.runtime.extension.api.annotation.Export;
import org.mule.runtime.extension.api.annotation.Extension;
import org.mule.runtime.extension.api.annotation.dsl.xml.Xml;

/**
 * This is the main class of an extension, is the entry point from which configurations, connection providers, operations
 * and sources are going to be declared.
 */
@Xml(prefix = "lauti")
@Extension(name = "Lauti")
@Export(resources = "holis.txt")
public class LautiExtension {
}
```

The above builds, it generates an `ExtensionModel` that can be consumed from Studio/Mozart (and it was generated from the maven archetype).
No seas más papista que el papa.

Reply:
>> No seas más papista que el papa.
@fernandezlautaro Are you talking about me?

Reply:
IMO that's wrong too, I'd fail now but backwards compatibility is a bitch

Reply:
m
e
t
e

e
l

p
r